src/pybind/mgr/orchestrator/module.py

```diff
@@ -352,25 +352,71 @@ def ukn(s):
 
             return HandleCommandResult(stdout=table.get_string())
 
+    @_cli_write_command(
+       'orch osd drivegroups show',
+       'name=format,type=CephChoices,strings=json|yaml,req=false',
+       'Get DriveGroups from mon store')
+    def _drivegroups_show(self, format='plain'):
+        # type: (Optional[str]) -> HandleCommandResult
+        completion = self.show_drivegroups()
+        self._orchestrator_wait([completion])
+        self.raise_if_exception(completion)
+        if format == 'json':
+            out = json.dumps(completion.result_str())
+        elif format == 'plain':
+            out = completion.result_str()
```

Comment:
maybe to yaml by default?

Reply:
I wasn't sure if the format actually matters.. We always return _stringified_ json, which is also compatible with yaml.. Maybe we drop the formatting altogehter?